opentuner/search/manipulator.py

```diff
@@ -245,7 +244,7 @@ def hash_config(self, config):
 
     def search_space_size(self):
         """estimate the size of the search space, not precise"""
-        return reduce(_ * _, [x.search_space_size() for x in self.params])
+        return reduce(lambda n: n * n, [x.search_space_size() for x in self.params])
```

Comment:
I think this should be:
```suggestion
        return reduce(lambda a, b: a * b, [x.search_space_size() for x in self.params])
```

Or `operator.mul`

Reply:
Fixed as suggested.